schema/rounds.json

```diff
@@ -1,31 +1,38 @@
 [
     {
         "name": "_id",
-        "type": "STRING"
+        "type": "STRING",
+        "description": "A unique identifier (opaque). Round number is kept in the `round` field."
     },
     {
         "name": "blockWasProposed",
-        "type": "BOOLEAN"
+        "type": "BOOLEAN",
+        "description": "Whether a block was proposed and executed in this round."
     },
     {
         "name": "epoch",
-        "type": "NUMERIC"
+        "type": "NUMERIC",
+        "description": "The epoch the round belongs to."
     },
     {
         "name": "round",
-        "type": "NUMERIC"
+        "type": "NUMERIC",
+        "description": "The number of the round."
     },
     {
         "name": "shardId",
-        "type": "NUMERIC"
+        "type": "NUMERIC",
+        "description": "The shard the round belongs to."
```

Comment:
not necessarily right, since all shards should always have the same round. A better description would be "The shard ID of the round entry. Each round will ideally produce 4 entries in this index."